cg/clients/freshdesk/freshdesk_client.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import requests
+
+from cg.clients.freshdesk.constants import EndPoints
+from cg.clients.freshdesk.models import Ticket
+
+LOG = logging.getLogger(__name__)
+
+
+class FreshdeskClient:
+    """Client for communicating with the freshdesk REST API."""
+
+    def __init__(self):
+        self.api_key = None
+        self.url = None
+
+    def init_app(self, url, api_key):
+        """Set up the client."""
+        self.url = url
+        self.api_key = api_key
```

Comment:
I really don't like the way of instantiating stuff. But to that seems to be way we do it in ext.py. I think a more intuitive way results in circular imports. Will evaluate in the coming PRs